### PR TITLE
style- card styling

### PR DESCRIPTION
**Pull request by ImrozNezami**
Cards
Vertical card
Card with badge
Card with dismiss button
Card with text overlay
Text card
 Horizontal card
Card with shadow

### CODE REVIEW

File: card.css [hunk 1]
```diff
@@ -0,0 +1,130 @@
+.card-container {
```
Comment (vaibhav-mahalle) [inline card.css:1]: content is overlapping in preview which you have provided.

File: card.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  height: 19rem;
+}
+.add-to-cart-btn {
+  border: none;
+  outline: 0;
+  padding: 0.6rem;
+  color: var(--white);
+  background-color: var(--secondary-color);
+  text-align: center;
+  cursor: pointer;
```
Comment (vaibhav-mahalle) [inline card.css:26]: you can give the button cursor pointer in utility class, so you wont have to write repeating code.

**Review by vaibhav-mahalle — APPROVED** [review]
suggested some minor changes. Please look into them.

File: card.css (hunk 1 above)
Comment (ImrozNezami) [reply card.css:1]: Preview is not right, my mistake should have added images of card.

File: card.css (hunk 2 above)
Comment (ImrozNezami) [reply card.css:26]: Okay

File: card.css [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+.vertical-card {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  position: relative;
+  max-width: 19rem;
+  text-align: center;
+  margin: auto;
+  margin-top: 0.6rem;
+  cursor: pointer;
+}
+.vertical-card img {
```
Comment (Sameony) [inline card.css:15]: directly manipulating the img tag is often discouraged
what u can do is put it inside a div tag and manipulate the div instead.

**Review by Sameony — APPROVED** [review]
good code, do study and use more utility classes.
another suggestion is given, do follow up and practice that when you have time

**Pull request merged**